config/jobs/kubernetes/kubeadm/kubeadm-presubmits.yaml

```diff
@@ -40,12 +49,16 @@ presubmits:
         securityContext:
           privileged: true
         resources:
+          limits:
+            cpu: 2
+            memory: 9Gi
           requests:
-            memory: "9000Mi"
-            cpu: 2000m
+            cpu: 2
+            memory: 9Gi
 
   # operator presubmits
   - name: pull-kubeadm-operator-verify
+    cluster: eks-prow-build-cluster
```

Comment:
this job is inactive as the folder doesn't exist (see filter). maybe we can just delete it for now.

Reply:
Sure, happy to delete! I'll send an update.

Reply:
Do you want to delete `pull-kubeadm-operator-verify-canary` also?

Reply:
yes, all operator jobs please

Reply:
Updated to remove operator jobs.